content/getting-started/index.md

```diff
@@ -1,12 +1,14 @@
 ---
 sidebar: getting-started
-next: "/getting-started/setting-up/"
+next: "/getting-started/registering/"
```

Comment:
This and excluding from the sidebar was intentional because I think most of the readers don't need that page and just gets in the way.

> Anyone can view the docs so it might be worth having a page that can be linked from the getting started page (e.g., "If you don't have an account yet, see ...").
> https://github.com/codewars/docs/issues/40#issuecomment-657816008

Reply:
I think it should at least have its entry in the sidebar, because having only inline link in one of articles makes this page very hidden and virtually impossible to find if anyone would, for some reason, need to. But it's okay for me remove both links too, if you want to 👍

Reply:
We can leave it in the sidebar because that doesn't get in the way and helps with discoverability. The search should reveal the page as well.